site/docs/v1/tech/core-concepts/_email-templates.adoc

```diff
@@ -0,0 +1,44 @@
+[.api]
+[field]#Email update# [optional]#Optional# [since]#Available since 1.30.0#::
+The email template to use when notifying a user that their email address has been updated.
+
+ifeval::["{page}" == "application"]
+[field]#Email verification# [optional]#Optional# [since]#Available since 1.27.0#::
+The email template to use when accounts are created to verify the User's email address.
+
+[field]#Email verified# [optional]#Optional# [since]#Available since 1.30.0#::
+The email template to use when notifying a user that email address has been verified.
+endif::[]
+
+[field]#Forgot password# [optional]#Optional#::
+The email template to use for the forgot password workflow that uses emails.
+
+[field]#Login Id duplicate on create# [optional]#Optional# [since]#Available since 1.30.0#::
+The email template to use when notifying a user that another user has attempted to register with the same username or email address.
+
+[field]#Login Id duplicate on update# [optional]#Optional# [since]#Available since 1.30.0#::
+The email template to use when notifying a user that another user has attempted to change their username or email address.
```

Comment:
This doesn't sound quite accurate, it sounds like another user is trying to change another user's email or username. 

 This is sent when another user is trying to change their email or username to that of the user we are sending this to. 

So maybe something like:

> The email template to use when notifying a user that another user has attempted to change their own email or username to a value in-use by the user.

Or something like that.